doc/book/router/uri-generation.md

```diff
@@ -13,41 +13,41 @@ the name:
 - If you call `route()` with no HTTP methods, the name is the literal path with
   no changes.
 
-  ```php
-  $app->route('/foo', $middleware); // "foo"
-  ```
```

Comment:
No. This was correctly indented; RTFD.org simply doesn't recognized fenced code blocks associated with bullet points. While we're initially publishing on RTFD.org, that is not the final location; the final location will use bookdown to render the documentation, and bookdown renders this correctly.